test/utils.spec.ts

```diff
@@ -2,7 +2,7 @@
 
 require('./testUtils.js');
 
-import * as utils from '../lib/utils';
+import * as utils from '../cjs/utils';
```

Comment:
I wonder if we should start importing TS files instead of compiled output. Maybe as a follow up.

Reply:
I agree this would be ideal! I've added a task to track this. When I tried to do this, I didn't have a lot of luck and ended up wrestling with mocha and a series of seemingly unrelated error messages for a few days 😓